src/hdpaths.spec.ts

```diff
@@ -0,0 +1,47 @@
+// tslint:disable:no-unused-expression
```

Comment:
Why is this necessary? Do we not have the tslint plugin?

Reply:
Jap, no chai/tslint plugin installed:

```
    "@types/bn.js": "^4.11.4",
    "@types/chai": "^4.1.7",
    "@types/koa": "^2.0.48",
    "@types/mocha": "^5.2.5",
    "chai": "^4.2.0",
    "mocha": "^5.2.0",
    "prettier": "^1.18.2",
    "tslint": "^5.18.0",
    "tslint-config-prettier": "^1.18.0",
    "tslint-immutable": "^6",
    "typescript": "~3.7"
```

Let's do that for eslint instead of tslint: https://github.com/iov-one/iov-faucet/issues/54